### DISCUSSION TITLE
Some time will crash

### DISCUSSION THREAD

**Discussion by Coloryr** [AvaloniaUI/Avalonia#14860] [Q&A]
## Describe the bug
When show and hide window, some time wil crash
I don't know what the exact triggering reason, but there is a probability of 5% -10% that the program will crash on its own.

## To Reproduce

1. hide window
2. show window
3. do many times

## Expected behavior
not crash.

## Screenshots
none

## Environment 

- OS: Windows
- Avalonia-Version: 11.0.7

## Additional context

![image](https://github.com/AvaloniaUI/Avalonia/assets/26276037/561c87c3-e309-4753-a619-17a39be9cf65)

**Comment by Coloryr** [comment]
My project https://github.com/Coloryr/ColorMC

**Comment by timunie** [comment]
I can do this without an issue: 

```cs
    public MainWindow()
    {
        InitializeComponent();

        DispatcherTimer timer =
            new DispatcherTimer(TimeSpan.FromMilliseconds(100), DispatcherPriority.Background, Tick);
        
        timer.Start();
    }

    private void Tick(object? sender, EventArgs e)
    {
        new Test().Show();
    }
```

So please try to strip down your sample as much as possible. Also worth to try nightly versions.

**Comment by Coloryr** [comment]
I am currently preparing for my application update and will try it out later.

**Comment by Coloryr** [comment]
The probability of this issue occurring is extremely low, and I may need some time to reproduce it

**Comment by timunie** [comment]
@Coloryr will make a Q&A of it now was we don't know how to reproduce. We can make it a bug report if sample is available.
